### PR TITLE
CME-269

### PR DESCRIPTION
**Pull request by wmoffett**
Remove Plasmic
Added dynamic routing for providers
Added API.tsx for providers via Canario API
First stab at content Parsing for madlib/augmentation.

### CODE REVIEW

File: components/magnolia/config.tsx [hunk 1]
```diff
@@ -7,9 +7,13 @@ import Paragraph from '@components/magnolia/components/Paragraph';
 import Expander from '@components/magnolia/components/Expander';
 import List from '@components/magnolia/components/List';
 import Item from '@components/magnolia/components/Item';
+import  TwoColumn from "@components/magnolia/components/TwoColumn";
+import  ThreeColumn from "@components/magnolia/components/ThreeColumn";
 
 const config = {
   componentMappings: {
+    'spa-lm:pages/basic': Basic,
```
Comment (dmacaring) [inline components/magnolia/config.tsx]: Well update these as we figure out naming for our base lm. Maybe do it now but def eventually

**Review by dmacaring — APPROVED** [review]

File: components/magnolia/config.tsx (hunk 1 above)
Comment (wmoffett) [reply components/magnolia/config.tsx]: Right! We can chat about it in the morning. I will make it happen before I create the repos as I need to add the light-module as well.

**Review by NInmanCaring — APPROVED** [review]

File: components/magnolia/EditablePage.tsx [hunk 2]
```diff
@@ -3,12 +3,13 @@ import React from 'react';
 import { EditablePage as MognoliaEditablePage } from "@magnolia/react-editor";
```
Comment (NInmanCaring) [inline components/magnolia/EditablePage.tsx]: 😂  
`MagnoliaEditablePage`

Comment (wmoffett) [reply components/magnolia/EditablePage.tsx]: lol

File: components/magnolia/config.tsx (hunk 1 above)
Comment (wmoffett) [reply components/magnolia/config.tsx]: updated to use base-lm

File: components/magnolia/EditablePage.tsx (hunk 2 above)
Comment (wmoffett) [reply components/magnolia/EditablePage.tsx]: Got it! lol

**Pull request merged**